src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -363,7 +363,7 @@ else
 	HARNESS_RUNNER="xharness"
 fi
 
-$__Command $HARNESS_RUNNER android test --instrumentation="net.dot.MonoRunner" --package-name="net.dot.$__Category" --app="$__TestBinaryBase/$__Category.apk" --output-directory=`pwd` --arg=entrypoint:libname=$(MsBuildProjectName).dll --expected-exit-code=100 -v
+$__Command $HARNESS_RUNNER android test --instrumentation="net.dot.MonoRunner" --package-name="net.dot.$__Category" --app="$__TestBinaryBase/$__Category.apk" --output-directory=`$HELIX_WORKITEM_UPLOAD_ROOT` --arg=entrypoint:libname=$(MsBuildProjectName).dll --expected-exit-code=100 -v
```

Comment:
@fanyang-mono I guess you want rather `"`?

```
--output-directory=`$HELIX_WORKITEM_UPLOAD_ROOT`
--output-directory="$HELIX_WORKITEM_UPLOAD_ROOT"
```

Not a big deal and can be fixed in the future but I guess we don't need to invoke a subshell for that

Reply:
Good catch! Will incorporate the the fix for future PR.